.travis.yml

```diff
@@ -119,6 +139,8 @@ script:
           -DUSE_SANITIZER=${USE_SANITIZER}
           -DBUILD_TESTING=on
           -G "${GENERATOR}" ../src
+  # execute coverity build, to let coverity integrate with the existing script code the following line is used to active the coverity build.
+  - if [[ -n $COVERITY_SCAN_PROJECT_NAME ]] ; then curl -s 'https://scan.coverity.com/scripts/travisci_build_coverity_scan.sh' | bash || true ; fi
```

Comment:
I really, really, REALLY, don't like it when people execute a script that they download over the internet without any verification of the contents. Apart from that, I also don't like to depend on a bash script. I believe integrating this into CMake would be better(?) It's a bit trickier though... We could also go for a CMake script that does more-or-less the same so people with Windows boxes can use the same logic. The `<build>/CMakeFiles/<cmake version>` subdirectory contains files named `CMakeCCompiler.cmake` and `CMakeCXXCompiler.cmake` we could use those to automatically configure Coverity based on the compiler path and compiler id, use most of the information already set by CMake to figure out the project name etc.